### DISCUSSION TITLE
Naming convention for branches

### DISCUSSION THREAD

**Discussion by hrishikesh-deshpande** [bhanujggandhi/HisaabKitab#8] [General]
Let's follow this :- https://medium.com/@abhay.pixolo/naming-conventions-for-git-branches-a-cheatsheet-8549feca2534

**Comment by bhanujggandhi** [comment]
@hrishikesh-deshpande @Mayush99 @avushek2305 @sharmamht19 
FYI! Add if any other suggestions

**Comment by bhanujggandhi** [comment]
Found a similar read:
https://graphite.dev/guides/git-branch-naming-conventions
